site/constants/index.ts

```diff
@@ -33,6 +34,42 @@ export const DISRUPTION_REASONS: {
     },
 ];
 
+export const DISRUPTION_SEVERITIES: {
+    value: Severity;
+    display: string;
+}[] = [
+    { value: Severity.unknown, display: "Unknown" },
+    { value: Severity.verySlight, display: "Very Slight" },
```

Comment:
I think we should use 'Very slight' (sentence case is the default for non-name values).

Reply:
done